usage/app/Global.scala

```diff
@@ -17,6 +17,10 @@ object Global extends WithFilters(CorsFilter, RequestLoggingFilter, new GzipFilt
   }
 
   override def onStart(app: Application) {
+    val crierReader = new KinesisStreamReader()
+
+    crierReader.start()
```

Comment:
As we are starting the worker before we've subscribed to the event stream, what's happening to events in the gap? Are they being queued for when the subscription starts or lost?